pkgs/build-support/docker/store-path-to-layer.sh

```diff
@@ -5,16 +5,29 @@ set -eu
 layerNumber=$1
 shift
 
+storePath="$1"
+shift
+
 layerPath="./layers/$layerNumber"
-echo "Creating layer #$layerNumber for $@"
+echo "Creating layer #$layerNumber for $storePath"
 
 mkdir -p "$layerPath"
-tar --no-recursion -rf "$layerPath/layer.tar" \
+
+# make sure /nix and /nix/store appear first in the archive
```

Comment:
Could you coment we are doing this to avoid file changes during the tar process.

Reply:
Done.